test/parallel/test-util-inspect.js

```diff
@@ -1950,6 +1950,81 @@ assert.strictEqual(util.inspect('"\'${a}'), "'\"\\'${a}'");
   );
 });
 
+// Verify that classes are properly inspected.
+[
+  // The whitespace is intentional.
+  // eslint-disable-next-line no-multi-spaces
+  [class   { }, 'class (anonymous) {}'],
+  [class extends Error { log() {} }, 'class (anonymous) extends Error {}'],
+  [class A { constructor(a) { this.a = a; } log() { return this.a; } }],
+  [class
+  // Random comments are part of the stringified result
+  äß extends TypeError {}, 'class äß extends TypeError {}'],
+  // The whitespace and new line is intended!
```

Comment:
there is also need a test for block comment

```suggestion
[class
  /*
  * Comments
  **/
  面包 extends Error {}, 'class 面包 extends Error {}'],
  // The whitespace and new line is intended!
```

Reply:
Thanks, adding some further tests did indeed find some further cases that were not handled so far. Thanks for the comment!